frameworks/cassandra/tests/config.py

```diff
@@ -89,6 +93,15 @@ def run_backup_and_restore(
     delete_data_job = get_delete_data_job(node_address=job_node_address)
     verify_deletion_job = get_verify_deletion_job(node_address=job_node_address)
 
+
+    # Ensure the keyspaces we will use aren't present.
+    try:
+        jobs.run_job(delete_data_job)
+    except:
```

Comment:
Fair. At this time having the stack trace give info (even if it's about an unexpected failure) can possibly help diagnose problems out of this scope though I suspect the extent of that help won't go too far.